src/components/AddBook.js

```diff
@@ -46,8 +46,8 @@ const AddBook = () => {
     <div className="add-book">
       <h1>ADD NEW BOOK</h1>
       <form>
-        <input type="text" placeholder="Book title" value={title} onChange={HandleTitleChange} required />
-        <input type="text" placeholder="Author" value={author} onChange={HandleAuthorChange} required />
+        <input type="text" className="input-title" placeholder="Book title" value={title} onChange={HandleTitleChange} required />
```

Comment:
- [ ] The navbar misses the horizontal line at the bottom
- [ ] The color of navbar, the white one stretches all the way
- [ ] There is a space between navbar and the books

Yours:
![image](https://github.com/0sugo/bookstore/assets/62705947/b2166037-21a0-4920-b504-ab6c5282a2ea)

Design:
![image](https://github.com/0sugo/bookstore/assets/62705947/12b0bfa6-3272-45be-86ce-56f179ad2b05)